libraries/Microsoft.Bot.Builder.Dialogs.Adaptive.Runtime/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -78,8 +78,16 @@ public static void AddBotRuntime(this IServiceCollection services, IConfiguratio
         internal static void AddBotRuntimeSkills(this IServiceCollection services, IConfiguration configuration)
         {
             var skillSettings = configuration.GetSection(SkillSettings.SkillSettingsKey).Get<SkillSettings>();
+            List<SkillConfigurationEntry> settings = configuration.GetSection("skill").Get<List<SkillConfigurationEntry>>();
```

Comment:
Nit: use var instead of explicit types (unless needed)
```suggestion
            var settings = configuration.GetSection("skill").Get<List<SkillConfigurationEntry>>();
```

Reply:
Few more nits on this line: Would be great to follow the same pattern where each setting class has a constant string with the config key. So you could move the "skill" constant to te SkillConfigurationEntry class and we can then reference it from tests and everywhere.

Reply:
Thanks for catching these, I've updated both to follow recommended practices.